entrypoint.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
+until mysql --protocol=socket -uroot -pAgil1234 -e 'select 1' > /dev/null; do echo "Waiting for mysql..."; sleep 2; done
```

Comment:
Unless you parse the MYSQL_HOST from the config, this will only work for docker-compose deployments.

Also, why not just ping it? 
`mysqladmin ping -h"$DB_HOST" --silent; `